manifests/plugin/gutterball.pp

```diff
@@ -0,0 +1,9 @@
+# gutterball plugin
+class katello::plugin::gutterball{
+    Class[ 'certs' ] ->
+    class { 'certs::gutterball': } ->
+    foreman::plugin { 'gutterball': } ->
+    class { 'gutterball':
```

Comment:
This will need to be `class { '::gutterball':` to prevent duplicate resource error. I think a simple spec test of this plugin manifest might be nice as it would have caught this error without having to run a full katello-installer test run.